src/main/java/com/starrocks/connector/flink/table/source/StarRocksExpressionExtractor.java

```diff
@@ -88,7 +88,10 @@ public String visit(CallExpression call) {
 
     @Override
     public String visit(ValueLiteralExpression valueLiteral) {
-        if (valueLiteral.getOutputDataType() == DataTypes.DATE() || valueLiteral.getOutputDataType().toString().equals("DATE NOT NULL")) {
+        if (valueLiteral.getOutputDataType().getLogicalType().getTypeRoot().equals(DataTypes.TIMESTAMP().getLogicalType().getTypeRoot()) || 
```

Comment:
Create a ref variable for valueLiteral.getOutputDataType().getLogicalType().getTypeRoot()